### DISCUSSION TITLE
Why not use Laravel Octane?

### DISCUSSION THREAD

**Discussion by ejunker** [tarampampam/laravel-roadrunner-in-docker#148] [Q&A] [answered]
I am still learning about Roadrunner. Why does this use spiral/roadrunner-laravel instead of Laravel Octane? What are the differences?

**Comment by tarampampam** [comment, marked as answer]
https://github.com/spiral/roadrunner-laravel/issues/39#issuecomment-842067918

**Reply by alekseyp** [reply]
https://github.com/roadrunner-server/roadrunner/issues/133#issuecomment-1270065614 😕
